src/app/users/users.component.ts

```diff
@@ -220,8 +222,21 @@ export class UsersComponent implements OnInit, OnDestroy, AfterViewInit {
     event.stopPropagation();
   }
 
+  deleteUserFromTeams(user) {
+    // //REPLACE THIS!
+    // this.couchService.findAll('teams').then(teamsDb => {
+    //   const docsWithUser = teamsDb.filter(doc => doc.userId === user._id)
+    //   this.couchService.bulkDocs('teams', docsWithUser.map(doc =>
+    //     this.couchService.delete('teams/' + doc._id + '?rev=' + doc._rev))
+    //   );
+    // })
+  }
+
   deleteUser(user) {
     const userId = 'org.couchdb.user:' + user.name;
+
+    this.deleteUserFromTeams(user);

```

Comment:
This line should be run as another element of the array in `forkJoin` on line 243